pkg/apic/provisioning/propertybuilder.go

```diff
@@ -187,6 +189,9 @@ func (p *schemaProperty) SetName(name string) TypePropertyBuilder {
 // SetLabel - sets the label of the property
 func (p *schemaProperty) SetLabel(label string) TypePropertyBuilder {
 	p.label = label
+	if p.label == OauthScopes {
+		p.uniqueItems = true
+	}
```

Comment:
this is too specific on how it expects the oath scopes, as mentioned above setting unique property should be controlled by the implementer of the builder and not limited to the label placed on the property